app/src/test/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryUpdateTest.kt

```diff
@@ -147,7 +147,7 @@ class BloodSugarEntryUpdateTest {
   }
```

Comment:
Could you reword the commit message to better reflect what is actually happening in the commit?

Ex: "Rename tests in `BloodSugarEntryUpdatedTest`".